### ISSUE TITLE
Plans to support NTS ?

### ISSUE THREAD

**Issue by leopignataro** [robertvazan/guerrillantp#16]
Dear authors of GuerrillaNtp,

Are there any plans to support Network Time Security (NTS), as specified on RFC 8915 ?

**Comment by robertvazan** [comment]
Nope. Sorry. I am low on spare time. Well-tested PR would be accepted though.

**Comment by leopignataro** [comment]
Dear Robert,

I completely understand the toll of pushing forward the development of open source libraries without the proper incentives.

My company, Lacuna Software, might be willing to sponsor the development of this feature, while keeping the library open source under the current license.

Would you be open to that?
